src/js/letters/containers/Main.jsx

```diff
@@ -26,10 +26,20 @@ export class Main extends React.Component {
     this.props.getAddressStates();
   }
 
+
+  appAvailability(lettersAvailability, addressAvailability) {
+    // If letters are available, but address is still awaiting response, consider the entire app to still be awaiting response
+    if (lettersAvailability === AVAILABILITY_STATUSES.awaitingResponse || addressAvailability === AVAILABILITY_STATUSES.awaitingResponse) {
```

Comment:
If you want to DRY this up a bit, you could always do a destructuring assignment first and then just use the `awaitingResponse` property in the logic:
`const { awaitingResponse } = AVAILABILITY_STATUSES`. Or you could just leave it alone :)

Reply:
good point, will add that!